scripts/installer.sh

```diff
@@ -0,0 +1,282 @@
+    print_error "Cloudlift package version mismatch."
+  fi
+  print_info "Cloudlift package installation test passed."
+}
+
+uninstall_cloudlift() {
+  print_info "Uninstalling Cloudlift package..."
+  if ! command -v "cloudlift" >/dev/null; then
+    print_info "Cloudlift package not found. Exiting..."
+    exit 0
+  fi
+  current_version=$(cloudlift --version | cut -d' ' -f3)
+  print_info "Current Cloudlift package version: ${current_version}"
+  confirm_message "Do you want to uninstall Cloudlift package?"
+
+  [ -d "${PACKAGE_DEST_DIR}/${PACKAGE_NAME}" ] && rm -rf "${PACKAGE_DEST_DIR}/${PACKAGE_NAME}"
+  [ -L "${PACKAGE_BIN_DIR}/${PACKAGE_NAME}" ] && rm "${PACKAGE_BIN_DIR}/${PACKAGE_NAME}"
+
+  if command -v "cloudlift" >/dev/null; then
+    print_error "Failed to uninstall Cloudlift package."
+  fi
+  print_info "Cloudlift package uninstalled successfully."
+}
+
+preflight_check() {
+  print_info "Running preflight checks..."
+  [ -z "${TERM}" ] && AUTO_APPROVE=true
+  for cmd in curl tar gzip; do
+    if ! command -v "$cmd" >/dev/null; then
+      print_error "$cmd is required."
+    fi
+  done
+
+  [ "${PACKAGE_VERSION}" = "latest" ] && get_latest_release
+
+  if command -v "cloudlift" >/dev/null; then
+    current_version=$(cloudlift --version | cut -d' ' -f3)
+    [ -z "${current_version}" ] && print_warning "The current Cloudlift did not install properly or is corrupted."
+    print_info "Cloudlift package already installed; version: v${current_version}"
+    confirm_message "Do you want to reinstall Cloudlift with version v${PACKAGE_VERSION}?"
+  fi
+}
+
+# ----------------------------------------
+# Main Execution
+# ----------------------------------------
+
+parse_arguments() {
+  while [ $# -gt 0 ]; do
+    case "$1" in
+    -h | --help)
+      usage
+      exit 0
+      ;;
+    -v | --version)
+      shift
+      PACKAGE_VERSION=$(echo "$1" | sed 's/^v//')
+      ;;
+    -u | --uninstall)
+      IS_UNINSTALL=true
+      ;;
+    -y | --auto-approve)
+      AUTO_APPROVE=true
+      ;;
+    *)
+      print_error "Invalid argument: $1"
+      ;;
+    esac
+    shift
+  done
+}
+
+main() {
+  parse_arguments "$@"
+
+  print_info "Cloudlift installation started..."
+  print_info "For OS: ${OS_TYPE}, Arch: ${ARCH_TYPE}, Version: ${PACKAGE_VERSION}"
+
+  check_sudo
+
+  if [ "${IS_UNINSTALL}" = true ]; then
+    uninstall_cloudlift
+  else
+    preflight_check
+    download_cloudlift
+    verify_checksum
+    unarchive_cloudlift
+    install_cloudlift
+    test_installation
+  fi
+}
+
+status_message() {
+  if [ $1 -eq 0 ]; then
+    version=$(cloudlift --version)
+    printf '
+            _                    _  _  _   __  _
+       ___ | |  ___   _   _   __| || |(_) / _|| |_
+      / __|| | / _ \ | | | | / _` || || || |_ | __|
+     | (__ | || (_) || |_| || (_| || || ||  _|| |_
+      \___||_| \___/  \__,_| \__,_||_||_||_|   \__|
+
+        \n'
+    print_info "Cloudlift package installed successfully; version: ${version}"
```

Comment:
`$ cloudlift --version` already prints version like this `cloudlift, version 2.0.6`
if you're printing `version: $version`, it might look like this:
`Cloudlift package installed successfully; version: cloudlift, version 2.0.6`

Reply:
it has beed fixed. now it shall show like `[INFO] Cloudlift package installed successfully; version: 2.0.6`